src/consul.js

```diff
@@ -16,14 +16,18 @@ async function get(url, fallback = null) {
     });
 
     if (consulData && consulData[0] && consulData[0].Value) {
-      return Buffer.from(consulData[0].Value, 'base64').toString();
+      return {
+        value: Buffer.from(consulData[0].Value, 'base64').toString()
```

Comment:
This and similar changes are due to the new slsv3 variable resolver engine having a different interface